### ISSUE TITLE
Rebirth 15+ heroes can't trigger an adventure

### ISSUE THREAD

**Issue by grayconcaves** [aikaterna/gobcog#99]
![devrebirth](https://user-images.githubusercontent.com/11259621/74681846-6454a280-51ff-11ea-91e9-a176bd8be02e.png)

I noticed this with a few users who finally hit Rebirth 15. Then I tested it out for myself using devrebirth. Users rb15+ can no longer use the =adventure command when they reach that rebirth level. Doesn't trigger any errors in the bot too.

EDIT: I went back to default theme and it finally worked, though it pulled an Ascended monster (which I hadn't made yet for my custom theme).

**Comment by Drapersniper** [comment]
It appears to be user error for not having all the required files.

However it should have notified you when attempting to change from the default there and not allowed to do it, @grayconcaves did you updated to 3.0.0 before or after changing to the custom there?

**Comment by grayconcaves** [comment]
No, there was no error for required files. I copied all of the files and modified them, so technically the bot didn't know that one of the files (in this case, Ascended Monsters) was empty. 

I had already updated to 3.0.0 and the latest Red version prior to changing the custom theme, so I believe it was because of the empty Ascended Monsters file. It's fixed now though that I've added a few entries in that file. Still worth reporting though for people with custom themes from the previous version?
